### ISSUE TITLE
Is there possible to replace "a few seconds" with "x seconds"?

### ISSUE THREAD

**Issue by krvstianeq** [iamkun/dayjs#1355]

**Comment by iamkun** [comment]
https://day.js.org/docs/en/customization/customization

**Comment by krvstianeq** [comment]
sadly it isn't understable for me :(

**Comment by iamkun** [comment]
You can check this as a reference https://day.js.org/docs/en/customization/relative-time

**Comment by krvstianeq** [comment]
What if I want to add one threshold key and string to the threshold key without disturbing the others?

**Comment by iamkun** [comment]
You can import the config object and update it accordingly.

**Comment by krvstianeq** [comment]
When I changed the threshold to the one from a documentation, the module is calculating time badly between day and day number two.

Example:
I wanted the module to check how many years would have passed between the date of March 4, 2016 and February 2, 2021. With the default threshold, it shows me a 5-year difference (it is still wrong) but when I changed the threshold to the one from a documentation, the module claims that the difference between this day and day two is only one year.
